mkosi/__init__.py

```diff
@@ -1675,7 +1684,10 @@ def make_image(state: MkosiState, skip: Sequence[str] = [], split: bool = False)
         definitions = state.workspace / "repart-definitions"
         if not definitions.exists():
             definitions.mkdir()
-            bootloader = state.root / f"efi/EFI/BOOT/BOOT{state.config.architecture.to_efi().upper()}.EFI"
+            if (a := state.config.architecture.to_efi()):
+                bootloader = state.root / f"efi/EFI/BOOT/BOOT{a.upper()}.EFI"
```

Comment:
Let's not be stingy and call it `arch` like above.
```suggestion
            if (arch := state.config.architecture.to_efi()):
                bootloader = state.root / f"efi/EFI/BOOT/BOOT{arch.upper()}.EFI"
```